transport/control.go

```diff
@@ -56,43 +56,33 @@ type windowUpdate struct {
 	increment uint32
 }
 
-func (windowUpdate) isItem() bool {
-	return true
-}
+func (*windowUpdate) isItem() {}
```

Comment:
Oh, I see. Both are already only used as pointers. Weird.

I'll defer to @iamqizhao. I agree that it's weird having the receiver type here be values, but it's also weird that the code is passing around pointers for these. I think for code clarity reasons I'd prefer to just change them all to be values. As a small bonus, putting an empty struct value like `flushIO` into an interface value is free, and doesn't require allocations like it is now:

```
transport/http2_client.go:                              t.controlBuf.put(&flushIO{})
transport/http2_client.go:                              case *flushIO:
transport/http2_server.go:                              t.controlBuf.put(&flushIO{})
transport/http2_server.go:                              case *flushIO:
```

Changing it to pointers makes it look like it has mutable state.